Makefile

```
@@ -59,7 +59,7 @@ HELM_CHART_LINT_ARGS_crossplane = --set nameOverride='',imagePullSecrets=''
 # Due to the way that the shared build logic works, images should
 # all be in folders at the same level (no additional levels of nesting).
 
-REGISTRY_ORGS = docker.io/crossplane
+REGISTRY_ORGS = docker.io/crossplane  xpkg.upbound.io/crossplane
```

@nullable-eth is this supposed to be a double space here?
```suggestion
REGISTRY_ORGS = docker.io/crossplane xpkg.upbound.io/crossplane
```

---

fixed